aiida/orm/groups.py

```diff
@@ -12,144 +12,85 @@
-                the groups created in the last past_days. Must be a datetime object.
-            :param node_attributes: if not None, must be a dictionary with
-                format {k: v}. It will filter and return only groups where there
-                is at least a node with an attribute with key=k and value=v.
-                Different keys of the dictionary are joined with AND (that is, the
-                group should satisfy all requirements.
-                v can be a base data type (str, bool, int, float, ...)
-                If it is a list or iterable, that the condition is checked so that
-                there should be at least a node in the group with key=k and
-                value=each of the values of the iterable.
-            :param kwargs: any other filter to be passed to DbGroup.objects.filter
-
-                Example: if ``node_attributes = {'elements': ['Ba', 'Ti'], 'md5sum': 'xxx'}``,
-                    it will find groups that contain the node with md5sum = 'xxx', and moreover
-                    contain at least one node for element 'Ba' and one node for element 'Ti'.
-
-            """
-            # Put everything in the kwargs
-            # Two warninigs:
-            #   1. Make sure to change the key if the variable name gets changed!
-            #   2. These are assuming that if the default gets passed in then that means the user didn't
-            #      specify it.  This a) precludes actually checking for something _being_ the whatever the
-            #      default is, and, b) means that we need to make the right check e.g. 'is not None' or is 'is'
-            #      depending on the type  (usually strings vs objects)
-            if name:
-                kwargs['name'] = name
-            if type_string:
-                kwargs['type_string'] = type_string
-            if id is not None:
-                kwargs['id'] = id
-            if uuid is not None:
-                kwargs['uuid'] = uuid
-            if nodes:
-                kwargs['nodes'] = nodes
-            if user is not None:
-                type_check(user, users.User)
-                kwargs['user'] = user.backend_entity
-            if node_attributes is not None:
-                kwargs['node_attributes'] = node_attributes
-            if past_days is not None:
-                kwargs['past_days'] = past_days
-
-            return [Group.from_backend_entity(group) for group in self._backend.groups.query(**kwargs)]
-
         def find(self, filters=None, order_by=None, limit=None):
             """Custom find method for Group to correctly map to backend type_string"""
             # We need to map type_string to type for now because the underlying ORM model
-            # for the current backends uses 'type' as the column name instead of type_string
+            # for the current backends uses 'type' as the column label instead of type_string
```

Comment:
Since we now migrated `type` to `type_string` this mapping is no longer necessary. In fact, I think that means the whole overriden find method can be removed

Reply:
yes, I deleted it and everything works